flow/connectors/kafka/kafka.go

```diff
@@ -75,7 +76,26 @@ func NewKafkaConnector(
 		kgo.WithLogger(kgoLogger(logger)),
 	)
 	if !config.DisableTls {
-		optionalOpts = append(optionalOpts, kgo.DialTLSConfig(&tls.Config{MinVersion: tls.VersionTLS13}))
+		tlsSetting := &tls.Config{MinVersion: tls.VersionTLS13}
```

Comment:
is TLS1.3 too bleeding edge? I'm not too familiar with the Kakfa ecosystem but iirc Confluent may not always support TLS 1.3

Reply:
Good point! I lowered the version to `tls.VersionTLS12`